module_fcst_grid_comp.F90

```diff
@@ -839,78 +779,78 @@ subroutine fcst_run_phase_2(fcst_comp, importState, exportState,clock,rc)
       type(ESMF_Clock)           :: clock
       integer,intent(out)        :: rc
 !
-!-----------------------------------------------------------------------
 !***  local variables
 !
-      integer                    :: i,j, mype, na, date(6), seconds
-      character(20)              :: compname
-
-      type(time_type)            :: restart_inctime
-      type(ESMF_Time)            :: currtime
+      integer                    :: mype, na, date(6), seconds
       integer(kind=ESMF_KIND_I8) :: ntimestep_esmf
       character(len=64)          :: timestamp
-!
-!-----------------------------------------------------------------------
-!
-      real(kind=8)   :: mpi_wtime, tbeg1
+      integer                    :: unit
+      real(kind=8)               :: mpi_wtime, tbeg1
 !
 !-----------------------------------------------------------------------
 !***********************************************************************
 !-----------------------------------------------------------------------
 !
       tbeg1 = mpi_wtime()
-      rc    = esmf_success
+      rc    = ESMF_SUCCESS
 !
 !-----------------------------------------------------------------------
 !
-      call ESMF_GridCompGet(fcst_comp, name=compname, localpet=mype, rc=rc)
+      call ESMF_GridCompGet(fcst_comp, localpet=mype, rc=rc)
```

Comment:
I think it's good to leave compname for debugging purpose

Reply:
It is unused. I can put it back, but my idea was to remove all unused variables so that we get clean compilation with `-Wall` compiler flag.